deft.js

```diff
@@ -115,29 +115,17 @@ deft.normalizeFilePaths = function(files) {
  * @return {string}
  *
  * @examples
- * deft.getUrl('lodash.js', ['lodash/lodash', 'lodash.js']);
- * // => 'https://raw.githubusercontent.com/lodash/lodash/master/lodash.js'
+ * deft.getInformationUrl('lodash.js', ['lodash/lodash', 'lodash.js']);
+ * // => 'https://api.github.com/repos/lodash/lodash/contents/lodash.js?ref=master'
  *
- * deft.getUrl('underscore.js', ['jashkenas/underscore', '1.6.0', 'underscore.js']);
```

Comment:
This doesn't look OK to me. Not sure if you're aware, but these examples are actual test cases using [autodoc](https://github.com/dtao/autodoc). It looks like your changes here break existing behavior, which is to support dependencies specified at arbitrary (non-GitHub) URLs. Or am I missing something?